Gemfile

```
@@ -283,7 +283,9 @@ group :web_socket, :manageiq_default do
 end
 
 group :appliance, :optional => true do
+  gem "irb",                            "=1.4.1",            :require => false # Locked to same version as the installed RPM rubygem-irb-1.4.1-142.module_el9+787+b20bfeee.noarch so that we don't bundle our own
   gem "manageiq-appliance_console",     "~>8.1",             :require => false
+  gem "rdoc",                                                :require => false # Needed for rails console
```

Super-mega-minor, but since the irb change and the rdoc change are related, can we bundle them together to something like

```suggestion
  gem "manageiq-appliance_console",     "~>8.1",             :require => false
  
  # Needed for console access in Ruby 3.1+
  gem "irb",                            "=1.4.1",            :require => false # Locked to same version as the installed RPM rubygem-irb-1.4.1-142.module_el9+787+b20bfeee.noarch so that we don't bundle our own
  gem "rdoc",                                                :require => false # Needed for rails console
```